R/interface.R

```diff
@@ -289,9 +289,16 @@ sourceExamples <- function(file){
 rcppoctave <- function(..., plugin = 'RcppOctave'){
 	if( !requireNamespace('inline') )
 		stop("Package 'inline' is required to define RcppOctave functions.")
-	#qlibrary(inline)
-	#rcpp(..., plugin = plugin)
-	cxxfunction(..., plugin = plugin)
+	#qlibrary('inline')
```

Comment:
I've tried to use `qlibrary` from `pkgmaker` without success. Checking `pkgmaker/NAMESPACE` I cannot find an `export` statement for this function.

For now, I'll have to comment the call to make this test run on my machine. Please advise, how to use `qlibrary` or how else you'd like this to be resolved, so that the PR can go ahead.